src/commands/deploy.ts

```diff
@@ -178,6 +182,11 @@ async function getDeployFsPath(): Promise<string> {
     return await workspaceUtil.selectWorkspaceFolder(ext.ui, workspaceMessage, (f: vscode.WorkspaceFolder) => getFuncExtensionSetting(deploySubpathSetting, f.uri.fsPath));
 }
 
+async function removeSubDeployFromFsPath(deployFsPath: string): Promise<string> {
```

Comment:
I'm not totally sold on this logic. Seems kinda hacky. Completely different approach: What if `ensureFolderIsOpen` returns the folder path and then you just use that folder path instead of deployFsPath for local git?

Reply:
Agreed.  I was looking for something just like this.

Should I rename the function to include AndReturn or do you think it'll be obvious enough?